Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -88,6 +90,37 @@ open class CarPlayNavigationViewController: UIViewController, BuildingHighlighti
         navigationService.router.continuousAlternatives
     }
     
+    func alternativesListTemplate() -> CPListTemplate {
+        var variants: [CPListSection] = []
+        let distanceFormatter = DistanceFormatter()
+        self.continuousAlternatives.forEach { alternative in
+            guard let title = alternative.indexedRouteResponse.currentRoute?.description else {
+                return
+            }
+            let distanceDelta = distanceFormatter.string(from: alternative.distanceDelta)
+            
+            let items: [CPListItem] = [CPListItem(text: "\(DateComponentsFormatter.travelTimeString(alternative.expectedTravelTimeDelta, signed: true)) / \(alternative.distanceDelta >= 0 ? "+":"")\(distanceDelta)",
```

Comment:
To avoid extra logic here, it should be possible to say something like:

```swift
numberFormatter.negativeFormat = "\(numberFormatter.positiveFormat) shorter"
numberFormatter.positiveFormat = "\(numberFormatter.positiveFormat) longer"
```

but with localized string formats instead of string interpolation.

Reply:
`negative` formats did not work as I expected (resulting string looked like "500 longer feet"), so I did it another way. It's a bit bulky but I could not find a cleaner way